### ISSUE TITLE
Checking Gender Details functionality

### ISSUE THREAD

**Issue by Abhishek19970** [woothemes/woocommerce#32064]
### Prerequisites

- [X] I have carried out troubleshooting steps and I believe I have found a bug.
- [X] I have searched for similar bugs in both open and closed issues and cannot find a duplicate.

### Describe the bug

verify that others option for gender available or not in drop down menu

### Expected behavior

other details must be available 
Male , Female ,Other

### Actual behavior

Other details not available

### Steps to reproduce

1.Login with admin credentials
2.Check for the gender details
3.Check wether other option available

### WordPress Environment

https://wcdemo.webkul.com/woocommerce-warehouse-management-system/
### Isolating the problem

- [ ] I have deactivated other plugins and confirmed this bug occurs when only WooCommerce plugin is active.
- [ ] This bug happens with a default WordPress theme active, or [Storefront](https://woocommerce.com/storefront/).
- [ ] I can reproduce this bug consistently using the steps above.

**Comment by DavidAnderson684** [comment]
Duplicate of previous erroneous report: https://github.com/woocommerce/woocommerce/issues/30170

**Comment by woocommercebot** [comment]
Hi @Abhishek19970,

  While our goal is to address all the issues reported in this repository,   GitHub should be treated as a place to report confirmed bugs only.

  The type of issue you submitted looks like a support request which may or may not reveal a bug once proper   troubleshooting is done.  In order to confirm the bug, please follow one of the steps below:

  - Review [WooCommerce Self-Service Guide](https://docs.woocommerce.com/document/woocommerce-self-service-guide/)   to see if the solutions listed there apply to your case;
  - If you are a paying customer of WooCommerce, contact WooCommerce support by   [opening a ticket or starting a live chat](https://woocommerce.com/contact-us/);
  - Make a post on [WooCommerce community forum](https://wordpress.org/support/plugin/woocommerce/)

  If you confirm the bug, please provide us with clear steps to reproduce it.

  We are closing this issue for now as it seems to be a support request and not a bug.   If we missed something, please leave a comment and we will take a second look.
